docs/guide/url_management.md

```diff
@@ -151,3 +151,22 @@ A transformation group consists of an array of commands (see [all available comm
 You can make use of pre-defined transformation groups.
 You can also add your own, with your own set of commands.
 To add commands to an existing group, provide the group name and list the commands you want to add.
+
+## Regenerating URL aliases
+
+You can use the `ezplatform:urls:regenerate-aliases` command to regenerate all URL aliases.
+Use is when:
+
+- you change URL alias configuration and want to regenerate old aliases
+- you encounter database corruption
+- you have content that for whatever reason does not have a URL alias
+
+Before applying the command, back up your database and make sure it is not modified while the command is running.
+
+``` bash
+bin/console ezplatform:urls:regenerate-aliases
+```
+
+Use an `--iteration-count` parameter to define how many Locations should be processed at once, to avoid too much memory use.
+
+After the command is applied, old aliases will redirect to the new ones.
```

Comment:
Hmm, at first I didn't notice that last line and was going to write it is missing ;) Can we advertise it better? (some border or something :D) 
It's one of the key requirements for the feature.